apps/console/src/features/identity-providers/components/wizards/iproov/iproov-authentication-provider-create-wizard.tsx

```diff
@@ -0,0 +1,472 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IdentityAppsError } from "@wso2is/core/errors";
+import { AlertLevels, IdentifiableComponentInterface, TestableComponentInterface } from "@wso2is/core/models";
+import { addAlert } from "@wso2is/core/store";
+import {
+    DocumentationLink,
+    GenericIcon,
+    Heading,
+    LinkButton,
+    PrimaryButton,
+    useDocumentation,
+    useWizardAlert
+} from "@wso2is/react-components";
+import { ContentLoader } from "@wso2is/react-components/src/components/loader/content-loader";
```

Comment:
Check whether we can omit `src` part from the import path.

```suggestion
import { ContentLoader } from "@wso2is/react-components/components/loader/content-loader";
```

Reply:
Tried this, but the path cannot be resolved then.

Reply:
Sorry, my suggestion is wrong. Could you please try the following?

`import { ContentLoader } from "@wso2is/react-components";`